tox.ini

```diff
@@ -27,7 +28,7 @@ commands=pidiff pushsource .
 [testenv:cov]
 usedevelop=true
 commands=
-	pytest --cov-report=html --cov=src {posargs}
+	pytest --cov-report=html --cov=src --cov=src/pushsource/_impl/utils ./ src/pushsource/_impl/utils {posargs}
```

Comment:
Can you please undo this change:

- it shouldn't be needed
- if it was needed, it would not be correct to make this change only for "cov".  You aren't only changing coverage-related arguments here, you're also asking it to look for tests under src/pushsource/_impl/utils and it wouldn't make sense to do that only for one testenv.